service/k8sclient/k8sclient.go

```diff
@@ -522,13 +522,23 @@ func (c *K8sClient) DeleteXtraDBCluster(ctx context.Context, name string) error
 	return nil
 }
 
+// ErrNotFound should be returned when referenced resource does not exist
+// inside Kubernetes cluster.
+var ErrNotFound error = errors.New("resource was not found in Kubernetes cluster")
+
 // GetXtraDBCluster returns an XtraDB cluster credentials.
 func (c *K8sClient) GetXtraDBCluster(ctx context.Context, name string) (*XtraDBCredentials, error) {
 	var cluster pxc.PerconaXtraDBCluster
 	err := c.kubeCtl.Get(ctx, string(perconaXtraDBClusterKind), name, &cluster)
 	if err != nil {
+		if errors.Is(err, kubectl.ErrNotFound) {
+			return nil, errors.Wrap(ErrNotFound, "cannot get XtraDb cluster")
```

Comment:
`cannot get XtraDB cluster credentials`

Reply:
I created const error template, used both for xtradb and mongo.